poll/translations/he/LC_MESSAGES/text.po

```diff
@@ -221,7 +221,7 @@ msgstr "נמצאה תשובה לא מוכרת '{answer_key}' עבור מפתח 
 
 #: poll/public/html/poll.html:35 poll/public/html/survey.html:53
 msgid "Submit"
-msgstr ""
+msgstr "הגשה"
```

Comment:
As you mentioned on the jira ticket, this translation is not showing up surprisingly.